cobalt/renderer/rasterizer/testdata/lottie_coverage/skottie-3d-2planes-expected.png.sha1

```diff
@@ -1 +1 @@
-de01c5b4870be2df6d53b46b8d92a749057eb5c8
+52e5ca2342411570fb3233a943b4a61c9137fc96
```

Comment:
Just a quick question. How do you get this?

Reply:
That means we update this expected file with the new result? Do we know what's changed comparing to the original result and if this is the expected behavior?

Reply:
Yes. See issues 156517158 and 226999079.